docs/conf.py

```diff
@@ -121,6 +122,34 @@ def check_dot_command():
 # The full version, including alpha/beta/rc tags.
 release = version
 
+
+iso_gpg_key: str
+oss_gpg_key: str
```

Comment:
The next step is to open some PRs on `inmanta-service-orchestrator` (to set `iso_gpg_key`) and `inmanta` (to set `oss_gpg_key`) in conf.py, one for each of iso{4.5.6} branch. 
Where do I find the right gpg key for each version/flavour ?

Reply:
We could choose to add the OSS key in this repo directly because it will never change (there is only 1 OSS version so only 1 repo). For iso I agree with your statement. You can find them on cloudsmith e.g. by opening the repo, then clicking "set me up" and picking Red Hat. Do you have a cloudsmith account?

Reply:
> Do you have a cloudsmith account?

Yes I do, thanks for the instructions :+1:

Reply:
found it there => https://code.inmanta.com/inmanta-infra/infra/-/blob/master/files/inmanta-and-epel-9-x86_64.cfg